plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveBucketHandle.java

```diff
@@ -16,6 +16,7 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.google.common.collect.ImmutableList;
+import io.trino.plugin.hive.metastore.SortingColumn;
```

Comment:
This is metastore-specific class and it could be better to use a dedicaed, metastore-agnostic representation here.

i cannot offer a concrete proposal here yet, though